src/pom.xml

```diff
@@ -1,54 +1,53 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Required for the pre-start script in the bosh world.
The pom installs the liqubase libraries 
This POM can be remove once bosh deployment goes away